src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensions.projitems

```diff
@@ -8,4 +8,305 @@
   <PropertyGroup Label="Configuration">
     <Import_RootNamespace>Microsoft.CodeAnalysis.Shared</Import_RootNamespace>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+  </ItemGroup>
```

Comment:
❗️ Only `<Compile>` items in **\*.projitems**. This file cannot be opened for editing in Visual Studio.

Reply:
Hmm, sounds reasonable, let me fix.

Reply:
Putting some more thought into this - any new project which includes this shared project will immediately get errors if we don't add this package reference which is needed for code inside this shared project. This can be a bigger issue once we ship this as a source NuGet package as per @tmat's suggestion. I will move this package reference into the individual projects for now, but something we should probably discuss.

Reply:
When it's in a nuget package the package itself can have a nuget dependency, so that should work out.